core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -974,6 +1002,41 @@ private ExprValue exprTimeToSec(ExprValue time) {
     return new ExprLongValue(time.timeValue().toSecondOfDay());
   }
 
+  /**
+   * UTC_DATE implementation for ExprValue.
+   * Returns in format `yyyy-MM-dd`.
+   *
+   * @return ExprValue.
+   */
+  private ExprValue exprUtcDate() {
+    return new ExprDateValue(exprUtcTimeStamp().dateValue());
+
+  }
+
+  /**
+   * UTC_TIME implementation for ExprValue.
+   * Returns in format `HH:mm:ss`.
```

Comment:
Same here.  I think this returns a ExprTimeValue, not a string with a format.

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/commit/59ef32f37ea15e9f44cae06574c3cb9f27db26e1.